modules/monitoring-configuring-the-monitoring-stack.adoc

```diff
@@ -136,9 +136,21 @@ The Prometheus config map component is called `prometheusK8s` in the `cluster-mo
 ====
 endif::openshift-dedicated,openshift-rosa[]
 
-. Save the file to apply the changes to the `ConfigMap` object. The pods affected by the new configuration are restarted automatically.
+. Save the file to apply the changes to the `ConfigMap` object.
 +
 [WARNING]
 ====
-When changes are saved to a monitoring config map, the pods and other resources in the related project might be redeployed. The running monitoring processes in that project might also be restarted.
+Different configuration changes to the `ConfigMap` object result in different outcomes:
+
+* The pods are not redeployed, therefore, there is no service outage.
```

Comment:
Change to: "The pods are not redeployed. Therefore, there is no service outage."